src/App.jsx

```diff
@@ -17,6 +18,7 @@ function App() {
       <Routes>
         <Route path='/' element={<Home />}></Route>
         <Route path='/articles/:id' element={<Article />}></Route>
+        <Route path='/articles/:articleId/comments' element={<CommentList />}></Route>
```

Comment:
Do we need a separate path. The comments are associated with an article as far as a user is concerned.